markdoc_nextjs_tags.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@markdoc/next.js/tags';
```

Comment:
I feel like we should just add this to `https://github.com/markdoc/next.js/blob/main/src/index.js`, and then we don't need to include that here.

Reply:
I think addint it to `index.d.ts` I think that would work. Do you want me to open a PR for that on `markdoc` repo?